masonry/src/event.rs

```diff
@@ -3,6 +3,7 @@
 
 //! Events.
 
+use std::panic;
```

Comment:
`panic!` is part of the prelude, and we don't seem to be using anything from the `panic` module here. Indeed, this still compiles removing this import.

I'm surprised this doesn't give a warning of any kind.